cmd/influx/cli/cli.go

```diff
@@ -182,7 +182,13 @@ func (c *CommandLine) Run() error {
 	for {
 		select {
 		case <-c.osSignals:
-			close(c.Quit)
+			// Attempt to gracefully exit
```

Comment:
I thought there was since I had done this previously, but this example seems to indicate there isn't.

```
package main

import "fmt"

func main() {
	for i := 0; i < 1000; i++ {
		ch := make(chan bool, 1)
		ch <- true
		closing := make(chan struct{})
		close(closing)
		select {
		case <-closing:
		case <-ch:
			fmt.Println("bad value", i)
			return
		}
	}
}
```

It prints `bad value` fairly quickly. So never mind.

It may just be more clear to duplicate the code from the `c.Quit` case rather than do some semi-convoluted for loop with a select statement. Whichever change is fine though.